js/card.js

```diff
@@ -0,0 +1,58 @@
+import { similarAds } from './data.js';
+
+const TYPES = {
+  flat: 'Квартира',
+  bungalow: 'Бунгало',
+  house: 'Дом',
+  palace: 'Дворец',
+  hotel: 'Отель',
+};
+
+const card = document.querySelector('#card-canvas');
+
+const cardTemplate = document.querySelector('#card')
+  .content.querySelector('.popup');
+
+const imgTemplate = cardTemplate.querySelector('img');
+
+const generatePhoto = function (elem) {
+  imgTemplate.src = `${elem}`;
+};
```

Comment:
элемент не может быть сырцом для картинки, сурс для кртинки - это url
```suggestion
const generatePhoto = function (url) {
  imgTemplate.src = url;
};
```